src/components/app-button/AppButton.tsx

```diff
@@ -12,7 +12,7 @@ interface AppButtonProps extends Omit<ButtonProps, 'size'> {
   disabled?: boolean
   component?: ElementType
   to?: string
-  size?: SizeEnum | null
+  size?: 'small' | 'medium' | 'large' | 'extraLarge' | 'xxl' | SizeEnum | null
```

Comment:
You can do it in a cleaner way
```suggestion
  size?: `${SizeEnum}` | SizeEnum | null
```

Reply:
And also, why did you estimate this issue for 2 SP? Did you have some problems finding out what caused this error?

Reply:
> And also, why did you estimate this issue for 2 SP? Did you have some problems finding out what caused this error?

I tried several other ways to fix the error so as not to change the `AppButton` on which many other files depended, but none of them were good. So I defined 2 SP as it took a while to find the best solution that would not cause errors in other files. Do you suggest estimating less?

Reply:
> You can do it in a cleaner way

Done.

Reply:
> > And also, why did you estimate this issue for 2 SP? Did you have some problems finding out what caused this error?
> 
> I tried several other ways to fix the error so as not to change the `AppButton` on which many other files depended, but none of them were good. So I defined 2 SP as it took a while to find the best solution that would not cause errors in other files. Do you suggest estimating less?

Actually, estimation of the bug should be done by you, and only you. But in my opinion, 2 SP is a little bit too much for a one line fix. Especially when we will replace AppButton with Button from design-system folder, so this fix is temporary. But, as I said before - it's up to you.